Cargo.toml

```
@@ -55,14 +55,14 @@ num-traits = "0.2"
 pretty_assertions = "0.6"
 proj = { version = "0.19", optional = true }
 quick-xml = "0.18"
-relational_types = "1"
+relational_types = { version = "2", git = "https://github.com/CanalTP/relational_types.git", branch = "collection-2.0.0" }
```

Must be changed before merging to
```suggestion
relational_types = "2"
```

---

done